cmd/keymasterd/certgen.go

```diff
@@ -385,6 +393,12 @@ func (state *RuntimeState) postAuthX509CertHandler(
 	if kubernetesHack {
 		organizations = userGroups
 	}
+	serviceMethods, err := state.getServiceMethods(targetUser)
```

Comment:
If the certificate is within the limits, I'd rather just issue it rather than force yet another configuration option to enable it. I basically view the inclusion of ServiceMethod data in GitDB as the way this is turned on.